types/react-native/index.d.ts

```diff
@@ -7001,6 +7001,7 @@ export interface NativeScrollEvent {
     layoutMeasurement: NativeScrollSize;
     velocity?: NativeScrollVelocity;
     zoomScale: number;
+    targetContentOffset?: NativeScrollPoint; // iOS-only
```

Comment:
Can we follow the convention and add a `@platform ios` label to a comment block above the property? This way it appears properly when using intellisense.

Reply:
(I guess this search https://github.com/facebook/react-native/search?q=targetContentOffset confirms that this prop only exists on ios...)

Reply:
Sure, I have changed it.